### PR TITLE
Init for DownsampleRegions

### PR DESCRIPTION
**Pull request by ffinfo**

### CODE REVIEW

File: src/test/resources/log4j.properties [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#
+# Biopet is built on top of GATK Queue for building bioinformatic
```
Comment (rhpvorderman) [inline src/test/resources/log4j.properties]: Old license

File: src/main/scala/nl/biopet/tools/downsampleregions/DownsampleRegions.scala [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+    logger.info("Done")
+  }
+
+  def argsParser = new ArgsParser(toolName)
+
+  def emptyArgs = Args()
+
+  def descriptionText: String =
+    """
```
Comment (rhpvorderman) [inline src/main/scala/nl/biopet/tools/downsampleregions/DownsampleRegions.scala:117]: Better to include the documentation at the top for readability.

**Review by rhpvorderman — CHANGES_REQUESTED** [review]
Some minor thinfs

File: src/test/resources/log4j.properties (hunk 1 above)
Comment (ffinfo) [reply src/test/resources/log4j.properties]: This should indeed be fixed, but this I think wrong in all tool for now

File: src/main/scala/nl/biopet/tools/downsampleregions/DownsampleRegions.scala (hunk 2 above)
Comment (ffinfo) [reply src/main/scala/nl/biopet/tools/downsampleregions/DownsampleRegions.scala:117]: Don't think I agree completely with this. Once someone is ready to look at the code they want to see the working code. This method are very good to have it there but there are just for generation of the code. What does miss here is maybe some scaladocs ;)

**Review by DavyCats — APPROVED** [review]

**Review by rhpvorderman — APPROVED** [review]

**Pull request merged**